app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -90,7 +90,7 @@ $include-html-global-classes: $include-html-classes;
 $font-family-sans-serif: "Nixie One", "Helvetica Neue", Helvetica, Roboto, Arial, sans-serif;
 // $font-family-serif: Georgia, Cambria, "Times New Roman", Times, serif;
 // $font-family-monospace: Consolas, "Liberation Mono", Courier, monospace;
-
+  $namespace : false;
```

Comment:
Line should be indented 0 spaces, but was indented 2 spaces<br>Variable names should be followed immediately by a colon